tests/log.cpp

```diff
@@ -132,4 +132,21 @@ TEST(LogBugStream, log_bug_should_be_aborted)
   EXPECT_DEATH(LOG(BUG) << content, output(__LINE__));
 }
 
-} // namespace bpftrace::test::log
+TEST(LogStreamBug, log_stream_bug_colorized)
```

Comment:
I dunno. This extra test (as well as any testing for the colorization of logs) feels like overkill.

Reply:
I won't block but the reason I think it's overkill is because log colorization is not business critical functionality for bpftrace. IMHO it's nice but not important enough to have and maintain unit tests for. Other people might feel differently. I just think there are always maintenance concerns when adding new code (unit tests are no exception).